go/stats/rates.go

```diff
@@ -73,16 +80,23 @@ type Rates struct {
 // If passing the special value of -1s as interval, we don't snapshot.
 // (use this for tests).
 func NewRates(name string, countTracker CountTracker, samples int, interval time.Duration) *Rates {
+	stack := debug.Stack()
+	if !strings.Contains(string(stack), "engine.go:385") && !strings.Contains(string(stack), "controller.go:77") {
+		log.Infof("NewRates(%v, %v, %v, %v, %s)", name, countTracker, samples, interval, debug.Stack())
+	}
```

Comment:
What happens when engine.go or controller.go are modified and the line numbers change. Will a test fail? Does it really matter? I guess this was added to aid in debugging. Maybe it can be removed altogether?

Reply:
There is one other Rates object that is leaky, which I was trying to debug, but is not easy to fix because it is nested in a mock package with no easy way to signal it. Will remove these lines before merging.